catalog/core/catalog-core-camelcomponent/src/main/java/ddf/camel/component/catalog/transformer/TransformerProducer.java

```diff
@@ -93,15 +95,15 @@ public void process(Exchange exchange)
 
     LOGGER.debug("MIME Type = {}", mimeType);
 
-    MimeTypeToTransformerMapper mapper = endpoint.getComponent().getMimeTypeToTransformerMapper();
+    Transform transform = endpoint.getComponent().getTransform();
 
-    if (mapper != null) {
-      LOGGER.debug("Got a MimeTypeToTransformerMapper service");
+    if (transform != null) {
+      LOGGER.debug("Got a Transform service");
 
-      metacard = transform(in, metacard, mimeType, transformerId, mapper);
+      metacard = transform(in, metacard, mimeType, transformerId);
     } else {
-      LOGGER.debug("Did not find a MimeTypeToTransformerMapper service");
-      throw new CatalogTransformerException("Did not find a MimeTypeToTransformerMapper service");
+      LOGGER.debug("Did not find a Transform service");
```

Comment:
i think log and throw is considered a code smell; we can probably remove this debug statement

Reply:
removed